package.json

```diff
@@ -7,9 +7,11 @@
     "lint": "eslint \"src/**/*.js\" \"test/**/*.js\" \"examples/**/*.js\"",
     "doc": "jsdoc -t node_modules/docdash --readme JSDOC.md -c jsdoc-config.json",
     "doclint": "npm run doc -- -t templates/silent",
-    "test": "npm run lint -- --max-warnings=0 && npm run build && npm run test-examples && npm run test-unit",
+    "test": "npm run lint -- --max-warnings=0 && npm run build && npm run test-examples && npm run test-with-coverage",
     "test-unit": "cross-env BABEL_DISABLE_CACHE=1 mocha --require babel-core/register test/*unit_test.js",
     "test-examples": "mocha -t 300000 --recursive test/examples",
+    "test-with-coverage": "nyc -n src -r html cross-env BABEL_DISABLE_CACHE=1 mocha --require babel-core/register test/*unit_test.js",
+    "deploy-coverage": "nyc report --reporter=text-lcov | coveralls",
```

Comment:
Do we need to configure our build process for that? ([the doc](https://docs.coveralls.io/supported-ci-services) is a bit unclear on that). Let's check this before merging (just to be sure not to break travis in the meantime). Maybe the best way is to test in your fork?

Reply:
As it works for open source projects, It already works for this PR :
https://coveralls.io/builds/17969819